src/style-guide/README.md

```diff
@@ -1257,6 +1257,47 @@ While attribute values without any spaces are not required to have quotes in HTM
 ```
 </div>
 
+### Avoid toRefs with props <sup data-p="b">strongly recommended</sup>
+
+**Access props using props.propName inside `setup`.**
```

Comment:
There's also [`toRef`](https://v3.vuejs.org/api/refs-api.html#toref). The documentation does recommend using that for grabbing individual `props` but it doesn't explicitly say 'because `toRefs` might not work'.